packages/app-page-layer/src/PageSelector.tsx

```diff
@@ -45,25 +52,30 @@ export const PageSelector: React.FC<PageSelectorProps> = () => {
     <div
       style={styles.page}
       onClick={() => {
-        setShowPageEditor(true);
+        setShowPageEditor(!showPageEditor);
```

Comment:
If the user clicks the the page is open it will close the selector UI and vice versa.